po/pt-BR.po

```diff
@@ -3448,15 +3448,15 @@ msgid "# Static and Constant Variables"
 msgstr "# Variáveis estáticas e constantes"
 
 #: src/basic-syntax/static-and-const.md:3
-msgid "Global state is managed with static and constant variables."
+msgid "Static and constants are two kind of variables with special constraints."
 msgstr "O estado global é gerenciado com variáveis estáticas e constantes."
 
 #: src/basic-syntax/static-and-const.md:5
 msgid "## `const`"
 msgstr "## `const`"
 
 #: src/basic-syntax/static-and-const.md:7
-msgid "You can declare compile-time constants:"
+msgid "Constants are values created at compile time. They are inherently more efficient because they don't need to be computed at runtime and the compiler can inline them and apply special optimizations that would not be possible with a larger set of values:"
```

Comment:
Ah, this illustrates the problem well: by changing the `msgid` field here, you're claiming that

> Constants are values created at compile time. They are inherently more efficient because they [...]

should be translated into Portuguese as

> Você pode declarar constantes de tempo de compilação:

I don't speak Portuguese, but that's clearly wrong :smile: